registry/exceptions.py

```diff
@@ -29,6 +29,13 @@ def __init__(self):
         super().__init__({})
 
 
+class InvalidMetadataException(CustomException):
```

Comment:
Is there a look up for INVALID_METADATA_EXCEPTION ?
As in is the error sent to the UI a different string or just INVALID_METADATA_EXCEPTION

Reply:
just string `INVALID_METADATA_EXCEPTION`